server/database/config/index.js

```diff
@@ -0,0 +1,7 @@
+const connection = require('./connection');
+const dbBuild = require('./build');
+
```

Comment:
Add This Function here so it's will build db when execute `npm run build:db`
```js
(async () => {
  try {
    await dbBuild();
    console.log('Build Successfully');
    connection.end();
  } catch (e) {
    console.log('Build Failed', e);
  }
})();
```

Reply:
done